data/data-pipeline/data_pipeline/score/score_narwhal.py

```diff
@@ -997,6 +997,33 @@ def _mark_donut_hole_tracts(self) -> pd.DataFrame:
             ]
         )
 
+    def _mark_tribal_dacs(self) -> None:
+        """Per the October 7th compromise (#1988),
+        tracts that are approx 100% tribal are Score N communities.
+        """
+        self.df[field_names.SCORE_N_COMMUNITIES] = np.where(
```

Comment:
might also be easier / make a bit more sense to just take the MAX of those two columns? and why overwrite SCORE_N_COMMUNITIES?

Reply:
no not a logical mistake at all! this makes sense to me! i am willy nilly with types